test/common/response_check.go

```diff
@@ -16,6 +16,7 @@ func CheckErr(t *testing.T, actualErr error, expErrNil bool, expErrorMsg ...stri
 	if expErrNil {
 		require.NoError(t, actualErr)
 	} else {
+		require.Error(t, actualErr)
```

Comment:
Why add this line? The following code checks whether the actual error message contains one of the expected error messages